store/conf/attrs/zimbra-attrs.xml

```diff
@@ -7912,7 +7912,7 @@ TODO: delete them permanently from here
 </attr>
 
 <attr id="1572" name="zimbraMobileForceSamsungProtocol25" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited,domainAdminModifiable" since="8.0.7">
-  <defaultCOSValue>FALSE</defaultCOSValue>
+  <defaultCOSValue>TRUE</defaultCOSValue>
```

Comment:
can this be applied in a patch?

Reply:
The target version is 10.1.0

Reply:
ok and will this be effective for already created cos in case of upgrade scenario or only for new installations?

Reply:
I think it will be effective also in case of upgrade.

Reply:
Default COS value won't get changed for existing one and so the inheritance for the account within the COS. It will need updating the COS for existing ones, for new ones it will work.